workspaces/orchestrator/plugins/orchestrator-common/src/openapi/openapi.yaml

```diff
@@ -121,7 +121,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/GetInstancesRequest'
+              $ref: '#/components/schemas/SearchRequest'
```

Comment:
please remove the `GetInstancesRequest` component definition too.

Reply:
done